lib/sudo-env/tests/env_tests.rs

```diff
@@ -132,6 +132,7 @@ fn create_test_context<'a>(sudo_options: &'a SudoOptions) -> Context<'a> {
         login: sudo_options.login,
         shell: sudo_options.shell,
         chdir: sudo_options.directory.clone(),
+        pid: std::process::id() as i32,
```

Comment:
I'm a bit worried that this context object will become this 'garbage heap' of variables that we need to pass around. Do we need to store the pid right now?

Maybe we should use the `sudo_system::Process::process_id()` instead of the std one? Mostly to make sure that pids are always of type `pid_t` (although the internals of that function could maybe use the std one instead).

Reply:
Hey so I agree with your worry about Context becoming this humongous struct but at the same time the alternative to pass this `pid` would be to do it as a parameter of the `run_command` function which is more or less the same. At the end I just want to be sure that I'm getting the PID of the top process and not something else by accident which goes very well with this `Context` type.

Reply:
About using sudo_system. Yeah i agree with that :)